release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -469,6 +469,26 @@ For more information, see xref:../registry/configuring_registry_storage/configur
 [id="ocp-4-17-machine-config-operator_{context}"]
 === Machine Config Operator
 
+[id="ocp-4-17-machine-config-operator-global-tls_{context}"]
+=== Control plane TLS security profiles supported by the MCO
+
+The Machine Config Operator and Machine Config Server now use the TLS security profile that is configured for the control plane components. For more information, see xref:../security/tls-security-profiles.adoc#tls-profiles-kubernetes-configuring_tls-security-profiles[Configuring the TLS security profile for the control plane].
```

Comment:
Maybe add the shortname in first-instance here since it's used in the heading:
```suggestion
The Machine Config Operator (MCO) and Machine Config Server now use the TLS security profile that is configured for the control plane components. For more information, see xref:../security/tls-security-profiles.adoc#tls-profiles-kubernetes-configuring_tls-security-profiles[Configuring the TLS security profile for the control plane].
```